R/add.R

```diff
@@ -120,16 +132,17 @@ rbind_at <- function(old, new, pos) {
 #' # add_column ---------------------------------
 #' df <- tibble(x = 1:3, y = 3:1)
 #'
-#' add_column(df, z = -1:1, w = 0)
+#' df %>% add_column(z = -1:1, w = 0)
+#' df %>% add_column(z = -1:1, .before = "y")
 #'
 #' # You can't overwrite existing columns
 #' \dontrun{
-#' add_column(df, x = 4:6)
+#' df %>% add_column(x = 4:6)
```

Comment:
These days I prefer `try()` over `\dontrun{}`.

Reply:
Oh, that is good to know. What is the best option?
`df %>% add_column(x = 4:6) %>% try()` or `try(df %>% add_column(x = 4:6))` 
Or is there another way?

Reply:
Changed to this ` try(df %>% add_column(x = 4:6))`